CHANGES.md

```diff
@@ -1,6 +1,42 @@
 Release Changes List
 ====================
 
+version 1.17.1
+==============
+
+Update to twentieth public release of Chapel, April 2018
+(see also changes below for 1.17.0)
+
+Bug Fixes
+---------
+* fixed a bug in which postinit() was not called for generic records
+* fixed a bug in which most launchers would not work with 'make install'
+* fixed a portability problem w.r.t. our processing of /proc/cpuinfo
+* fixed a bug in which LinearAlgebra.sparse.eye() would unintentionally promote
+
+Cray-specific Changes
+---------------------
+* reduced contention for polling threads for CHPL_COMM=ugni
+* fixed a bug w/ unregisterable pre-existing memory segments for CHPL_COMM=ugni
```

Comment:
I wouldn't characterize the old behavior as a "bug".  How about "improved how we handle unregisterable memory segments for CHPL_COMM=ugni"?  (It fits in 72 columns.)

Reply:
See if you like the rewording I just pushed (tried to avoid the first-person pronoun)

Reply:
Yep, that's great.  I wondered about "we" but decided it was okay after finding some instances of "our".  Granted it's not quite the same, though.